.github/workflows/release.yml

```diff
@@ -76,12 +76,54 @@ jobs:
           cp ./semgrep-ubuntu/artifacts.tar.gz.sha256 semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
           gh release --repo returntocorp/semgrep upload ${{ steps.get_version.outputs.VERSION }} semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
           rm semgrep-${{ steps.get_version.outputs.VERSION }}-ubuntu-16.04.tgz.sha256
+      - uses: actions/download-artifact@v3
+        with:
+          name: rule-schema-json
+          path: rule_schema.json
+      - uses: actions/download-artifact@v3
+        with:
+          name: rule-schema-yaml
+          path: rule_schema.yaml
+      - name: Upload Schema Files
+        id: upload-semgrep-schema-files
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release --repo returntocorp/semgrep-interfaces upload ${{ steps.get_version.outputs.VERSION }} rule_schema.json
+          gh release --repo returntocorp/semgrep-interfaces upload ${{ steps.get_version.outputs.VERSION }} rule_schema.yaml
       - name: Publish Release
         id: publish_release
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           gh release --repo returntocorp/semgrep edit ${{ steps.get_version.outputs.VERSION }} --draft=false
+      - name: Publish Release Semgrep Interfaces
+        id: publish_release_semgrep_interfaces
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          gh release --repo returntocorp/semgrep-interfaces edit ${{ steps.get_version.outputs.VERSION }} --draft=false
```

Comment:
Auth needed (see above)

Reply:
Will the token for this repo not work for the other?